src/passes/implicitConversionToExplicit.ts

```diff
@@ -626,13 +626,17 @@ function handleConcatArgs(node: FunctionCall, ast: AST) {
 
 function handleAbiEncodeArgs(args: Expression[], ast: AST) {
   args.forEach((arg) => {
-    const type = safeGetNodeType(arg, ast.compilerVersion);
+    const type = safeGetNodeType(arg, ast.inference);
     if (type instanceof StringLiteralType) {
       insertConversionIfNecessary(arg, new BytesType(), arg, ast);
     } else if (type instanceof IntLiteralType) {
       assert(arg instanceof Literal);
       const signed = BigInt(arg.value) < 0;
       insertConversionIfNecessary(arg, new IntType(256, signed), arg, ast);
+    } else if (type instanceof TupleType) {
+      arg.typeString = `uint8[${type.elements.length}] memory`;
```

Comment:
Not causing any test to fail. Hardcoding memory doesn't look like a good idea though